Manito/Manito/Presentation/Scene/ParticipateRoomScene/ViewModel/ChooseCharacterViewModel.swift

```diff
@@ -24,13 +24,13 @@ final class ChooseCharacterViewModel: BaseViewModelType {
     
     private let roomIdSubject = PassthroughSubject<Result<Int, ChooseCharacterError>, Never>()
     
-    private let participateRoomService: ParticipateRoomService
+    private let usecase: ParticipateRoomUsecaseImpl
```

Comment:
p1;

```suggestion
    private let usecase: ParticipateRoomUsecase
```

여기엔 프로토콜로 추상적으로 선언해야 할 것 같네요!

Reply:
[8367f3c](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/548/commits/8367f3c67e2af0f1e6c114ef85cf81f57bf57bec)

놓친 부분이었네요 감사합니다!